src/openrct2/peep/Guest.cpp

```diff
@@ -5901,12 +5901,7 @@ bool Guest::ShouldFindBench()
         return false;
     }
 
-    if (!GetNextIsSurface() && !GetNextIsSloped())
-    {
-        return true;
-    }
-
-    return false;
+    return !GetNextIsSurface() && !GetNextIsSloped();
```

Comment:
Old notation is easier to read.

Reply:
`!(GetNextIsSurface() || GetNextIsSloped())`?

Reply:
I'm fairly sure @Gymnasiast referred to the if-block. Personally, I disagree; `!GetNextIsSurface() && !GetNextIsSloped()` is a succinct and equivalent expression.